CHANGES.rst

```diff
@@ -11,6 +11,8 @@ New Features
 
 - Plots within plugins can now be popped-out into their own windows. [#2254]
 
+- Add launcher to select config and require --layout argument when launching standalone [#2257]
```

Comment:
I thought you didn't want to mention the launcher?
```suggestion
- Add launcher to select config and require --layout argument when launching standalone. [#2257]
```

Reply:
I didn't want to mention it in the docs. I feel like I can't really avoid it in the changelog. The docs feel more like an advertisement to the users, an the changelog feels less of that. 

Changelog only really shows the launcher exists, but doesn't say anything about how to use access it, which is where we'd run into trouble

Reply:
Change log is meant for the *users*. If they see it and they are excited about it but they cannot figure it out, then we end up getting help calls anyway. But then again, I am not sure if any of our users are reading this, so 🤷 .